static/entrypoint.js

```diff
@@ -437,20 +447,22 @@ async function prepare(control, data) {
           return resolve(config);
         }
 
-        // If the page wants a subscription, they're listening to the Firebase config data.
+        // If the page wants a subscription, they're listening to the Firebase config data PLUS
+        // the page's params (in "data" key). This only happens for modvil.
```

Comment:
I assume `modvil` to be a the model for the villiage.

Reply:
It's the scene name (in static/scenes/modvil). It's "Modular Village".

We've redone the village a few times so I gave them (for better or worse) new names whenever we did it. The last one was "dorf".